.travis.yml

```diff
@@ -20,8 +20,8 @@ matrix:
 
     - language: swift
       os: osx
-      osx_image: xcode11.3
-      env: SWIFT=5.1
+      osx_image: xcode11.6
+      env: SWIFT=5.2
```

Comment:
Do we intend to be compatible with multiple Swift versions or only 5.2 exclusively?

Reply:
5.2 exclusively. PiranhaSwift won't be backward compatible with the 5.1 version anymore.